lib/scripts/seed_test_db.ts

```diff
@@ -66,26 +68,28 @@ async function seedUsers(
       createdAt: createdAt.toISOString(),
       updatedAt: updatedAt.toISOString(),
       lastLoggedIn: updatedAt.toISOString(),
-      // make half the users patients, and the other half staff locally
-      role: userNumber < midpoint ? Role.Patient : Role.Staff
+      // make half the users pharmacists, and the other half staff locally
```

Comment:
do we need patients as well tho for testing despite not having a patient UI

Reply:
Oh Good Point. Okay will make some patients too!